tests/data/generic/sizes/test.sizes

```diff
@@ -0,0 +1 @@
+chr7	159138663
```

Comment:
Use this file instead: https://github.com/nf-core/modules/blob/master/tests/data/genomics/sarscov2/general/test.genome.sizes

Reply:
Hi @drpatelh, thank you for the review! I have changed the test data to use the sizes of sars-cov2 and the bed12 with the sars-cov2 coordinates.

Reply:
The test_ucsc_bed12tobigbed now uses the sarscov2/geneneral/test.genome.sizes